happi/item.py

```diff
@@ -65,12 +66,20 @@ class MyItem(HappiItem):
                                  enforce_doc='This must be a number')
     """
 
-    def __init__(self, doc=None, optional=True, enforce=None,
-                 default=None, enforce_doc=None):
+    def __init__(
+        self,
+        doc: Optional[str] = None,
+        optional: Optional[bool] = True,
+        enforce: Optional[Any] = None,
+        default: Optional[Any] = None,
+        enforce_doc: Optional[str] = None,
+        include_default_as_kwarg: Optional[bool] = True
```

Comment:
I think this deserves a docstring call-out above that makes it clear where/when to use it